### DISCUSSION TITLE
Intel 和Nvidia/PGI fortran 设置module路径不起作用

### DISCUSSION THREAD

**Discussion by xiangyi-wang** [xmake-io/xmake#2285] [Q&A] [answered]
` set_values("fortran.moduledir", "$(buildir)/mod")`

gfortran是正常输出到mod路径下  Intel fortran 和NVIDIA  fortran 没有生效，是`fortran.moduledir`的写法问题吗

**Comment by waruqi** [comment, marked as answer]
目前没 ifort 等环境测试，暂时做了限制
 https://github.com/xmake-io/xmake/blob/ac4e75aa8863be23f984a44a3b05f5855251441a/xmake/rules/fortran/xmake.lua#L24

**Reply by xiangyi-wang** [reply]
意思是这里的限制是在windows平台吗，我的环境是wsl2，ubuntu20.04

**Reply by waruqi** [reply]
只要是 win 上的，目前都限制了，包括 wsl

**Reply by xiangyi-wang** [reply]
好的 谢谢
